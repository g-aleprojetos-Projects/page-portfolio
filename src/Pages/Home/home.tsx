import React from 'react';
import * as S from './home.styles';

export const Home = () => {
  return (
    <S.Container>
      <S.ContainerImagem>
        <S.Imagem />
      </S.ContainerImagem>
    </S.Container>
  );
};
