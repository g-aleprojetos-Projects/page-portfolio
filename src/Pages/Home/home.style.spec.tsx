import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import 'jest-styled-components';
import * as S from './home.styles';

describe('Home.styles', () => {
  test('Container DEVE ser igual ao snapshot passando true em "backgroundBlack"', () => {
    const container = renderer
      .create(<S.Container backgroundBlack={true} />)
      .toJSON();
    expect(container).toMatchSnapshot();
  });
  test('Container DEVE ser igual ao snapshot passando false em "backgroundBlack"', () => {
    const container = renderer
      .create(<S.Container backgroundBlack={false} />)
      .toJSON();
    expect(container).toMatchSnapshot();
  });
});
