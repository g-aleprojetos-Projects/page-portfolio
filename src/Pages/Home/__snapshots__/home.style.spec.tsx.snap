// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.styles Container DEVE ser igual ao snapshot passando false em "on" 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100vw;
  height: 100vh;
  background-color: #d3d3d3;
}

<div
  className="c0"
  on={false}
/>
`;

exports[`Home.styles Container DEVE ser igual ao snapshot passando true em "on" 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100vw;
  height: 100vh;
  background-color: #360033;
}

<div
  className="c0"
  on={true}
/>
`;
